website/addons/mendeley/static/widget-cfg.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var $osf = require('osfHelpers');
+var citations = require('../../../static/js/citations');
+var CitationGrid = require('../../../static/js/citationGrid');
+
+require('./citations_widget.css');
+
+////////////////
+// Public API //
+////////////////
+
+function CitationsWidget(gridSelector, styleSelector) {
```

Comment:
This seems to be the public API for `CitationGrid`. Any reason not to put this there?